testdata/get-no-gomod.txt

```diff
@@ -0,0 +1,25 @@
+# Get a module that has no go.mod file. Gohack should add the
+# go.mod file for us.
+cd repo
+go get rsc.io/sampler@v1.2.1
+env GOHACK=$WORK/gohack
+gohack get rsc.io/sampler
+
+exists $GOHACK/rsc.io/sampler/go.mod
+gohack undo rsc.io/sampler
+exists $GOHACK/rsc.io/sampler/go.mod
+
+gohack get rsc.io/sampler
+stdout '^rsc.io/sampler => .*/gohack/rsc.io/sampler$'
```

Comment:
What's the significance of doing ` gohack get` twice?

Why not check the `stdout` first time around?